manager/ui/war/plugins/api-manager/html/api/api-impl.html

```diff
@@ -104,6 +104,19 @@
               </div>
             </div>
 
+            <div style="margin-top: 10px">

```

Comment:
Please avoid inline styling for new elements and add a new CSS class/id

Reply:
I just copied and paste the following HTML
`<div class="" style="margin-top:10px;" ng-show="updatedApi.parsePayload">` =S